pkgs/tools/security/flare-floss/default.nix

```diff
@@ -1,45 +1,110 @@
 { lib
-, python3
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, nose
+, pytest
+, pytestCheckHook
 , fetchFromGitHub
+, tabulate
+, vivisect
+, viv-utils
+, pydantic
+, halo
+, tqdm
+, pytest-cov
+, pytest-instafail
+, pytest-sugar
+, pyyaml
+, types-pyyaml
+, types-tabulate
 }:
+let
+  # required for networkx 2.5.1
+  decorator_4_4_2 = buildPythonPackage rec {
+    pname = "decorator";
+    version = "4.4.2";
 
-python3.pkgs.buildPythonPackage rec {
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-46YvBSAXJEDKDcyCN0kxk4Ljd/N/FAoLme9F/suEv+c=";
+    };
+  };
+
+  # flare-floss requires this exact version (newer versions are incompatible)
+  networkx_2_5_1 = buildPythonPackage rec {
+    pname = "networkx";
+    version = "2.5.1";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-EJzVhcrEEpf3EQPDxCrG73N58peI61TLdRvlpmO7I1o=";
+    };
+
+    propagatedBuildInputs = [ decorator_4_4_2 setuptools ];
+    checkInputs = [ nose pytest pytestCheckHook ];
+  };
```

Comment:
Consider defining global derivation version alias. Tho I am not sure what the correct way of doing this